### PR TITLE
Use the accept language parameter

### PR DESCRIPTION
**Pull request by jobjen02**
The $AcceptLanguage parameter didn't get used. So I updated the generator and the baseClient to use the parameter.

### CODE REVIEW

**Comment by jobjen02** [comment]
Please let me know if there is anything that i'm missing!

File: src/BaseClient.php [hunk 1]
```diff
@@ -423,6 +423,10 @@ private function prepareAndExecuteRequest(string $method, string $url, array $op
             'Authorization' => sprintf('Bearer %s', $this->accessToken->getToken()),
         ];
 
+        if (! empty($options['language'])) {
```
Comment (WoutervanderLoopNL) [inline src/BaseClient.php:426]: - for code standard remove the space after `!` 
- specify a comment: `// pass through Accept-Language header`

**Review by WoutervanderLoopNL — CHANGES_REQUESTED** [review]
Great :+1:

File: src/BaseClient.php (hunk 1 above)
Comment (jobjen02) [reply src/BaseClient.php]: Updated, thx!

**Comment by jobjen02** [comment]
> I cannot find in the documentation why this is needed and how it works. Can you show me what this is for?


In this endpoint for example: https://api.bol.com/retailer/public/redoc/v10/retailer.html#tag/Product-Content/operation/get-catalog-product. It expects the language as an header.

**Comment by casperbakker** [comment]
Gotcha, thanks. Seems like it was only half-supported before.

**Comment by casperbakker** [comment]
Included in this release: https://github.com/picqer/bol-retailer-php-client/releases/tag/v10.2.2

**Pull request merged**